src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -44,6 +44,7 @@ internal sealed partial class Http2Connection : HttpConnectionBase, IDisposable
         private int _pendingWindowUpdate;
         private long _idleSinceTickCount;
         private int _pendingWriters;
+        private bool _hasAfterPendingWriteFrames;
```

Comment:
Nit: can we call this something like "_needsFlush"? Or if we want to be more explicit, "_flushWhenNoMoreWriters" or something like that? Something that indicates a flush is needed?

Reply:
_needsFlush is too generic, in that if there any data in the buffer, it needs a flush.  But I can rename it to something like _flushWhenNoMoreWriters.